### PR TITLE
Movie page

### PR DESCRIPTION
**Pull request by jervalles**
Movie page is ready on mobile.

### CODE REVIEW

File: frontend/src/components/MoviePageJoker.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import MoviePage from "./MoviePage";
+import movieList from "../DataBaseMovie";
+
+const MoviePageJoker = () => (
+  <div className="MoviePageJoker">
```
Comment (thalice) [inline frontend/src/components/MoviePageJoker.jsx]: Change your className or erase it (doesn't return to any CSS), thx:)

Comment (jervalles) [reply frontend/src/components/MoviePageJoker.jsx]: children ClassName deleted

File: frontend/src/components/MoviePageJoker.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import MoviePage from "./MoviePage";
+import movieList from "../DataBaseMovie";
+
+const MoviePageJoker = () => (
+  <div>
+    {movieList
+      .filter(toto => {
```
Comment (LoicBrassart) [inline frontend/src/components/MoviePageJoker.jsx]: Toto ! :scream:

File: frontend/src/components/MoviePageJoker.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import MoviePage from "./MoviePage";
+import movieList from "../DataBaseMovie";
+
+const MoviePageJoker = () => (
```
Comment (LoicBrassart) [inline frontend/src/components/MoviePageJoker.jsx]: Why this name ?

**Pull request merged**